src/users/interfaces/revoke-all-sessions-for-user-options.interface.ts

```diff
@@ -0,0 +1,3 @@
+export interface RevokeAllSessionsForUserOptions {
+  id: string;
```

Comment:
I think we should call this `userId`

Reply:
implementing + updating docs accordingly